tests/test_load_waypoint_name_to_coordinates_map.py

```diff
@@ -3,6 +3,7 @@
 """
 import pathlib
 
+from modules import waypoint
```

Comment:
Move below line 7.

Reply:
Not sure what is meant by this. If it means to switch the places of the module imports, I thought I was told on the last PR to put "Shorter names before longer names"?

Reply:
Imports are sorted alphabetically, but if the 1st characters are the same, the shorter is above the longer. Example:
```py
from modules import name
from modules import name_long
from modules.something import aaa
from modules.thing import aa
from modules.thing.other import b
```